blender/LilySurfaceScrapper/Scrappers/CgbookcaseScrapper.py

```diff
@@ -73,9 +88,15 @@ def fetchVariant(self, variant_index, material_data):
             'Roughness': 'roughness',
             'Metallic': 'metallic',
         }
-        for m in v.xpath(".//a[@class='directdownload']"):
-            map_name = m.xpath("div[@class='downloadable']/text()")[0].strip()
+        for m in v.xpath(".//a"):
             map_url = "https://www.cgbookcase.com" + m.attrib['href']
+
+            temp = map_url[map_url.find("K_") + 2:-4].split("_")[-1::-1]
+            map_name = " ".join(map_url[map_url.find("K_") + 2:-4].split("_")[-1::-1]).title()
+            map_name = temp[0].capitalize()
+            # Add another folder for front / back textures
+            material_data.name += "/" + temp[1].capitalize() if double_sided else ""
```

Comment:
Does not feel right to do this within a loop, it will accumulate the name of all the maps. I think it is better to account for the double-sided variation by modifying the `variant_name` above. ;)

Reply:
Oh, you're right, didn't look at the output.

My intention was that it would create a seperate folder for front a backsides.
I'll change it so that it will just put all of them in the same folder. Much simpler.

Reply:
I think your intention is good, but this is not the place to realize it. Just add something like this before the loop:

```python
if double_sided:
    variant_name += "/front" if variant_index < len(variants) / 2 else "/back"
```

Reply:
Do we have a general naming scheme that textures should adhere to?

Reply:
I'd say `source/base_name/variant[/subvariant]`

Reply:
> I'd say `source/base_name/variant[/subvariant]`

Hmm, I'd prefer `source/base_name[/front or back]/variant` because it would mean less folders, I think